linux/just_git_airgap_repo.bsh

```diff
@@ -121,13 +121,21 @@ JUST_HELP_FILES+=("${BASH_SOURCE[0]}")
 # #. ``source setup.env``
 # #. ``just git clone-airgap-repo`` - Clone submodules recursively from the new mirror
 #
+# .. note::
+#
+#   Limitations - There are a few limitations with a mirrored repository:
+#
+#     1. While the mirrored repository is a full fledged git repo, care must be taken to ensure subsequent transfers are successful. Ideally, the mirrored repository should be considered read-only; however, all that is technically required is that the transferred branches remain read-only---additional branches can be created in the repo and merged to as needed.
+#
+#     #. If a submodule is removed from the repository, then checking out an old version of the repository prior to its removal and trying to update its submodules with the git_update-submodules target will fail. This is because, before the submodule can be cloned, it must first be re-configured to point to the URL specified by repo_map.env. However, the old submodule will not be found in this map because :func:`git_mirror git_mirror_main`, and by extension this plugin, only create mirrors of submodules **currently** in a specific branch. Even inspecting the history of the .gitmodules file and including all submodules ever tracked by a branch could be problematic if, for example, the identity of a submodule (at a particular path) changed at some point (this is probably fairly unlikely, but would still be a problem).
```

Comment:
```suggestion
#    Limitations - There are a few limitations with a mirrored repository:
#
#    1. While the mirrored repository is a normal git repository, care must be taken to ensure subsequent (incremental) mirrors are successful: specifically, the transferred branches must remain read-only. However, additional branches/tags can be created as long as their names won't clash with those from the host repository.
#
#    #. :func:`git_mirror git_mirror_main`, and by extension this plugin, does not mirror all submodules that have ever been part of the repo, only those from a specific branch/SHA/tag you specify (master by default). Consequently, checking out another version of the repository with a different version of the .gitmodules file in which a submodule has been deleted or renamed may cause the :cmd:`git_clone-airgap-repo` to fail because the submodule's remote URL could not be re-configured to point to the mirror.
```